test/known_issues/test-vm-data-property-writable.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/10223
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+const context = vm.createContext({});
+
+const code = `
+   Object.defineProperty(this, 'foo', {value: 5});
+   Object.getOwnPropertyDescriptor(this, 'foo').writable;
```

Comment:
I tried, but I don't think ```assert.deepStrictEqual()``` works as intended...
I tested on a positive example and it throws an error:

```
const desc = vm.runInContext(code, context);
//desc: { value: 5, writable: true, enumerable: true, configurable: true }

const foo = {value: 5, writable: true, configurable: true, enumerable: true};
assert.deepStrictEqual(desc, foo); // throw new assert.AssertionError
```